Core/GDCore/Extensions/Builtin/AdvancedExtension.cpp

```diff
@@ -72,6 +72,18 @@ void GD_CORE_API BuiltinExtensionsImplementer::ImplementsAdvancedExtension(
       .AddParameter("trueorfalse", "Should the condition be true or false?")
       .MarkAsAdvanced();
 
+  extension
+      .AddAction("SetReturnObject",
+                 _("Set object return value"),
+                 _("Returns the currently selected objects. Those objects will be the ones selected for the next actions, conditions, subevents etc."),
+                 _("Set objects to return to _PARAM0_"),
```

Comment:
We should probably find a name that implies that we're not really "returning" anything but rather "pick objects _PARAM0_ as the result of this condition"?
(speaking of which, this should usually only happen in a condition, even if in theory actions can also use this).

Reply:
I named it return because I think that is both more useful and intuitive. This action also sets the return value to true, so it does return "something". The logic behind it is that if you have no objects to pick, then a previous condition had 0 objects so it should be false, so you are not returning any objects and want your condition not to be true either. If you did pick objects in your previous conditions, then you change the picked objects and want the condition to return true since some objects are picked.